ui/main.qml

```diff
@@ -125,7 +125,7 @@ StatusWindow {
     Connections {
         id: windowsOsNotificationsConnection
         enabled: false
```

Comment:
This is initialized later on with:
```qml
                if (Qt.platform.os === Constants.windows) {
                    windowsOsNotificationsConnection.enabled = true
                }
```

Better put it as a binding here?

Reply:
```suggestion
        enabled: Qt.platform.os === Constants.windows
```

Reply:
> This is initialized later on with:
> 
> ```qml
>                 if (Qt.platform.os === Constants.windows) {
>                     windowsOsNotificationsConnection.enabled = true
>                 }
> ```
> 
> Better put it as a binding here?

okay, but problem was in the module instantiation :) I will add